plugins/org.eclipse.xtext/src/org/eclipse/xtext/workspace/IProjectConfig.java

```diff
@@ -0,0 +1,19 @@
+/*******************************************************************************
+ * Copyright (c) 2015 itemis AG (http://www.itemis.eu) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *******************************************************************************/
+package org.eclipse.xtext.workspace;
+
+import java.util.Set;
+
+import org.eclipse.emf.common.util.URI;
+
+public interface IProjectConfig {
```

Comment:
Since we have different understandings of project on different platforms, we should describe what a project is on the intf